Setting another environment variable in AWS + Updating README

ghostcodekc opened the pull request:
Added environment variable 'CLOUD_NAME' in Lambda correlating with the Cloudinary cloud name, so it's easier for user setup. 

Added in information about locales for English for MomentJS.

ghostcodekc committed:
Add files via upload

ghostcodekc committed:
Add files via upload

olemartin commented on warlog.js:
```diff
@@ -41,45 +41,52 @@ exports.handler = async (event, context) => {
                     .unix(battle.utcTime)
-                    .locale('nb')
-                    .tz('Europe/Oslo')
+                    .locale('en')
```
You should have this as environment variables as well

olemartin replied:
these.. :)

olemartin commented on warlog.js:
```diff
@@ -97,6 +104,20 @@ exports.handler = async (event, context) => {
     return results;
 };
 
+const equalDeck = (warDeck, otherDecks) => {
```
I don't understand why this is diffing, but I figure it out

ghostcodekc replied:
I'm not sure, I downloaded the current master branch when I worked on the environment variables. Let me know if you figure something out :)

olemartin reviewed (commented):
Great work! Would be great if you could look at my comments

olemartin committed:
Merge branch 'master' into master

ghostcodekc committed:
Add files via upload

- Changed time zone and locale to environmental variables. 
- Changed the cloudinary cloud name to an environmental variable.
- Updated readme.md

ghostcodekc commented:
Sorry it took me so long. I changed the time zone and language locale to environment variables in AWS. :)

olemartin commented:
Thank you so much!

I had some other changes going, and I couldn't figure out how to merge these two changes in a good way. So I copied the changes from your pull request and added it to my commit.
So your changes are not lost even if I close the pull request.
Thanks again 💯 👍
